lib/react_test_utils.dart

```diff
@@ -190,18 +219,37 @@ external bool isElement(dynamic object);
 @JS('React.addons.TestUtils.isElementOfType')
 external bool _isElementOfType(dynamic element, dynamic componentClass);
 
-/// Returns true if [element] is a ReactElement whose type is of a
-/// React componentClass.
+/// __Deprecated. Will be removed in the `5.0.0` release.__ Use [isElementOfTypeV2] instead.
+///
+/// Returns `true` if [element] is a [ReactElement]
+/// that is of the `type` of the provided [componentFactory].
+@Deprecated('4.4.0')
```

Comment:
Just saw this... These should be the version they're to be removed in, not the version they were deprecated in.